python-ecosys/aiohttp/aiohttp/aiohttp_ws.py

```diff
@@ -143,7 +143,7 @@ async def handshake(self, uri, ssl, req):
         headers["Host"] = f"{uri.hostname}:{uri.port}"
         headers["Connection"] = "Upgrade"
         headers["Upgrade"] = "websocket"
-        headers["Sec-WebSocket-Key"] = key
+        headers["Sec-WebSocket-Key"] = key.decode("utf-8")
```

Comment:
Please use `str(key, "utf-8")`, because the `decode()` method is not always available.